testData/org/rust/lang/core/resolve/fixtures/nested_child_mod/inner/child.rs

```diff
@@ -0,0 +1,2 @@
+fn foo() {}
+fn bar() {}
```

Comment:
Have to use two functions here because of https://github.com/alexeykudinkin/intellij-rust/issues/119